src/plugins/intel_cpu/src/nodes/rnn.cpp

```diff
@@ -896,6 +910,8 @@ void RNN::execute(dnnl::stream strm) {
         {DNNL_ARG_DST_LAYER,     dst_data_mem->GetPrimitive()},
     };
 
+    getRuntimeScratchPad()->setScratchPad(args, scratchpad_md);
+
```

Comment:
Just wondering why did we place all these primitive args initialization (not only `setScratchPad`) into the `execute` call? This stuff creates some overhead on every execute call.

Reply:
I don't know about other primitive args, but there is reason for scratchpad since it's shared by all nodes in a graph, and it may get resized on node-by-node basis.

if we only set it in prepareParams(), suppose node1 is executed before node2 and node2 causes a resize in first inference (which would invalidated node1's scratchpad but that's OK since node1 has finished execution), and in second inference no shape changes thus node1  didn't call prepareParam() and it would accessing an wild pointer.

Reply:
Ok, then let's live this as it is.